changelogs/fragments/module_respawn.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- Module API - new module_respawn API allows modules that need to run under a specific Python interpreter to respawn in place under that interpreter
+- Module API - libselinux-python is no longer required for basic module API selinux operations
```

Comment:
This is awesome :tada:
It will help users like me who have been bumping into selinux/dnf/yum/apt errors every now and then for the past years, thanks ! 

Before landing, would it be possible to provide an item in the changelog with the list of modules that are impacted ? It's good news worth calling out :)